ipinfo/main.go

```diff
@@ -65,6 +68,8 @@ func main() {
 		err = cmdRandIP()
 	case cmd == "cache":
 		err = cmdCache()
+	case cmd == "config":
```

Comment:
We should update the default help message, and anywhere else this should appear.